frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -422,8 +512,72 @@ export const ClusterServiceVersionList: React.SFC<ClusterServiceVersionListProps
   );
   const NoDataEmptyMsg = () => <MsgBox title="No Operators Found" detail={noDataDetail} />;
 
-  return (
+  // remove duplicate global operator entries
+  const filteredData =
+    data.length > 0
+      ? data.filter((source) => {
+          const kind = referenceFor(source);
+          if (source == null) {
+            return null;
+          }
+          if (referenceForModel(SubscriptionModel) === kind) {
+            return source;
+          }
+          if (referenceForModel(ClusterServiceVersionModel) === kind) {
+            if (source.metadata.annotations?.['olm.operatorGroup'] === 'global-operators') {
+              return source.metadata.namespace === 'openshift-operators' ? source : null;
+            }
+            return source;
+          }
+          return null;
+        })
+      : [];
+
+  return ns === ALL_NAMESPACES_KEY ? (
+    <Table
+      data={filteredData as any[]}
+      {...rest}
+      aria-label="Installed Operators"
+      Header={AllProjectsClusterServiceVersionTableHeader}
+      Row={(rowArgs: RowFunctionArgs<ClusterServiceVersionKind | SubscriptionKind>) => (
+        <InstalledOperatorTableRow
+          obj={rowArgs.obj}
+          index={rowArgs.index}
+          rowKey={rowArgs.key}
+          style={rowArgs.style}
+          catalogSources={catalogSources.data}
+          subscriptions={subscriptions.data}
+        />
+      )}
+      EmptyMsg={NoOperatorsMatchFilterMsg}
+      NoDataEmptyMsg={NoDataEmptyMsg}
+      virtualize
+      customSorts={{
+        managedNamespace: (obj) => {
+          const olmOperatorGroup = obj.metadata?.annotations?.['olm.operatorGroup'];
+          let olmManagedNamespace;
+          if (olmOperatorGroup === 'global-operators') {
```

Comment:
I don't think this is right for native k8s. I believe it's more correct to just return `All Namespaces` when the annotation is missing from the CSV.